contrib/qubes/doc/uefi_capsule_update.md

```diff
@@ -0,0 +1,151 @@
+# UEFI capsule update
+
+The qubes-fwupd handle the UEFI capsule update under several conditions.
+The fwupd uses ESRT tables to read GUID, and that causes trouble when the OS
+is running under a hypervisor. The Xen does not pass the ESRT tables to
+paravirtualized dom0, so the Qubes is not able to provide sysfs information.
+More information you can find it this thread:
+
+https://patchwork.kernel.org/patch/11715901/
+
+## Requirements
+
+### Qubes OS
+
+You need [Qubes R4.1](https://openqa.qubes-os.org/tests/10816/asset/iso/Qubes-4.1-20200726-x86_64.iso)
+to use the UEFI capsule update.
+
+### Hardware
+
+Make sure that your hardware has available firmware updates in the [LVFS](https://fwupd.org/)
+
+### Custom dom0 kernel
+
+You have to build a custom linux kernel with
+[patch](../misc/0017-esrt-Add-paravirtualization-support.patch) the issue
```

Comment:
this seems like a dead URL to me being a relative link

Reply:
Fixed

Reply:
Still seems like the broken URL to me.